_episodes/41.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: episode
+title: "41: Trino puts on its Hudi"
+date: 2022-10-27
+tags: hudi apache datalake table format sql
+youtube_id: ""
+wistia_id: ""
+sections: 
+   - title: "Releases"
+     desc: "396 to 402"
+     time: 0
+   - title: "Concept of the episode"
+     desc: "What is Hudi?"
+     time: 687
+   - title: "Concept"
+     desc: "Why Iceberg over other formats?"
+     time: 1010
+   - title: "Concept"
+     desc: "Metadata catalogs"
+     time: 2140
+   - title: "Concept"
+     desc: "Branching, tagging, and auditing, oh my!"
+     time: 2634
+   - title: "Concept"
+     desc: "The Puffin file format"
+     time: 3053
+   - title: "Concept"
+     desc: "Trino Iceberg connector updates"
+     time: 3698
+   - title: "PR of the episode"
+     desc: "PR 13111: Scale table writers per task based on throughput"
+     time: 4297
+   - title: "Demo"
+     desc: "DML operations on Iceberg using Trino"
+     time: 4531
+---
+
+## Hosts
+
+ * Brian Olsen, Developer Advocate at [Starburst](https://starburst.io)
+ ([@bitsondatadev](https://twitter.com/bitsondatadev))
+ * [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate at 
+ [Starburst](https://starburst.io)
+
+## Guests
+
+ * Ryan Blue, creator of Iceberg and CEO at 
+ [Tabular](https://tabular.io) ([@rdblue](https://github.com/rdblue))
+ * Sam Redai, Developer Advocate at [Tabular](https://tabular.io)
+ ([@samuelredai](https://twitter.com/samuelredai))
+ * [Tom Nats](https://www.linkedin.com/in/tomnats), Director of Customer Solutions at [Starburst](https://starburst.io)
+
+## Register for Trino Summit 2022!
+
+Trino Summit 2022 is coming around the corner! This **free** event on November
+10th will take place in-person at the Commonwealth Club in San Francisco, CA or
+can also be attended remotely!  If you want to present, the 
+[call for speakers](https://sessionize.com/trino-summit-2022/) is open until
+September 15th. 
+
+[You can register for the conference](https://www.starburst.io/info/trinosummit/)
+at any time. We must limit in-person registrations to 250 
+attendees, so register soon if you plan on attending in person!
+
+## Releases 394 to 395
+
+Official highlights from Martin Traverso:
+
+[Trino 396]({{site.url}}/docs/current/release/release-396.html)
+
+* 
+
+
+Additional highlights worth a mention according to Cole:
+
+* 
```

Comment:
```suggestion
* The new Hudi connector is worth mentioning twice. It was in the works for a while, and we're really excited it's here.
* Trino 396 added support for version 3 of the Delta Lake writer, than Trino 401 added support for version 4, so we've jumped from 2 to 4 since the last time you saw us!
* There have been a ton of fixes to table and column comments across a wide variety of connectors.
```